packages/evolution-common/src/services/baseObjects/README.md

```diff
@@ -79,6 +79,147 @@ A type representing an individual audit entry.
+
+### Key Attributes
+
+#### `preData`
+
+- **Purpose**: Stores custom prefilled data from external sources
+- **Type**: `Record<string, unknown>` (flexible object structure)
+- **Format**: Survey-specific; each project defines its own structure
+- **Available on**: All survey objects
+- **Use case**: Store any imported data that doesn't directly map to standard object attributes but may be useful for validation, display, or later processing
+
+#### `preGeography`
+
+- **Purpose**: Stores the original geographic location from the imported data
+- **Type**: `GeoJSON.Feature<GeoJSON.Point>`
+- **Available on**: Place and other location-based objects (Home, usual work and school places)
+- **Use case**: Preserve the original imported location separate from the potentially corrected/validated `geography` attribute, allowing comparison between imported and final declared locations
+
+### Data Flow
+
+1. **Import**: External data is imported and stored in the `sv_interviews_prefill` table using the `importPreFilledResponses` task
+2. **Mapping**: The import task maps CSV columns to survey object paths (e.g., `home.address`, `home.postalCode`)
+3. **Geography Conversion**: If coordinates are provided, they are converted to GeoJSON points and stored in both `preGeography` (original) and `geography` (editable)
+4. **Zone Calculation**: Intersecting geographic zones are pre-calculated and stored with the prefilled data
+5. **Interview Creation**: When an interview starts, the system retrieves prefilled data using `getByReferenceValue()` based on the reference field (e.g., access code)
+6. **Population**: Survey objects are created with both standard attributes and prefilled data in the `preData` attribute
+7. **Validation**: Users can validate or modify prefilled data during the interview
+8. **Preservation**: Original prefilled data remains in `preData` and `preGeography` for auditing and comparison
+
+### Database Structure
+
+The `sv_interviews_prefill` table contains:
+- `reference_field` (string, unique, indexed): The key used to lookup prefilled data (e.g., access code)
+- `responses` (JSON): The prefilled response data in the format:
+  ```typescript
+  {
+      [path: string]: {
+          value: unknown;
+          actionIfPresent?: 'force' | 'doNothing'
+      }
+  }
+  ```
+
+### Import Task
+
+The `importPreFilledResponses` task (`evolution-backend/src/tasks/importPreFilledResponses.task.ts`) provides a template for importing prefilled data from CSV files.
+
+**Important**: This task is designed as an **example template** that must be customized for each survey project. If your CSV has different column names or structure, you must create a custom import task in your survey project.
```

Comment:
is a template, but also a good default that _can_ (not "must") be customized for each survey project.